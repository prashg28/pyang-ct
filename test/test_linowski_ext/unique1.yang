 module unique1 {

   namespace "urn:org:example:xml:ct:unique1";
   prefix "u";

   import complex-types {prefix "ct"; }

   ct:complex-type  c1;


   ct:instance node1 {
       ct:instance-type c1;
   }

   leaf node1 {
       type int8;
   }

   ct:instance-list node1 {
       ct:instance-type c1;
   }

}
