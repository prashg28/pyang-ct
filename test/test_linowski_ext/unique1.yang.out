unique1.yang:15: error: DUPLICATE_CHILD_NAME
unique1.yang:19: error: DUPLICATE_CHILD_NAME
unique1.yang:19: error: KEY_REQUIRED
